ironfish/src/workerPool/serializableMessage.ts

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import bufio from 'bufio'
+import { Serializable } from '../common/serializable'
+
+export enum WorkerMessageType {
+  BoxMessage = 'boxMessage',
+  CreateMinersFee = 'createMinersFee',
+  CreateTransaction = 'createTransaction',
+  GetUnspentNotes = 'getUnspentNotes',
+  MineHeader = 'mineHeader',
+  Sleep = 'sleep',
+  SubmitTelemetry = 'submitTelemetry',
+  TransactionFee = 'transactionFee',
+  UnboxMessage = 'unboxMessage',
+  VerifyTransaction = 'verifyTransaction',
+}
+
+export abstract class SerializableWorkerMessage implements Serializable {
```

Comment:
What do you think about `WorkerMessage`?

```suggestion
export abstract class WorkerMessage implements Serializable {
```

Reply:
That sounds good; changed it.